starter-code/app.js

```diff
@@ -1,5 +1,36 @@
+'use strict';
+
 const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+//Configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+
+
+//Routes
+// retrieve a random chuck joke
+app.get('/', (req, res, next) => {
+    console.log('Get Working');
+    res.render('index');
```

Comment:
Here, in fact, you want to display the same thing that in /random (not only the index but with a joke), for doing that:
```
res.redirect('/random');
```